README.md

```diff
@@ -23,6 +24,10 @@ To download LiteXpansion please click here: [Download LiteXpansion](https://theb
   </a>
 </p>
 
+## :wrench: Config
+
+- `nerf-other-addons` - Enable this option will nerf Energized Solar Generator and panels in InfinityExpansion, otherwise all machines in LiteXpansion will consume double energy. This option is enabled by default.
+
```

Comment:
Move this to the configuration section of the readme at the bottom

Reply:
its still not in the configuration section
![image](https://user-images.githubusercontent.com/44972470/144248728-19e40595-5292-49b0-b509-dc60ccd299fe.png)